src/lib/viewers/controls/sidebar/ThumbnailsToggle.tsx

```diff
@@ -4,15 +4,17 @@ import './ThumbnailsToggle.scss';
 
 export type Props = {
     onThumbnailsToggle?: () => void;
+    isThumbnailsOpen?: boolean;
```

Comment:
We typically try to keep our props, types, variables, etc. alphabetized when possible.

Reply:
Updated!